pkgs/os-specific/linux/joycond-cemuhook/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, python3Packages, fetchFromGitHub}:
+
+python3Packages.buildPythonPackage {
+  pname = "joycond-cemuhook";
+  format = "pyproject";
+  version = "unstable-2023-9-3";
```

Comment:
```suggestion
  version = "unstable-2023-08-09";
```

Reply:
My bad I have no clue how I got that date wrong. I guess I just read it wrong.